zarr/tests/test_core.py

```diff
@@ -21,7 +23,14 @@
 from zarr.errors import PermissionError
 from zarr.compat import PY2
 from zarr.util import buffer_size
-from numcodecs import Delta, FixedScaleOffset, Zlib, Blosc, BZ2
+from numcodecs import (Delta, FixedScaleOffset, Zlib, Blosc, BZ2, MsgPack, Pickle,
+                       Categorize, JSON)
+
+
+# needed for PY2/PY3 consistent behaviour
+if PY2:  # pragma: py3 no cover
+    warnings.resetwarnings()
+    warnings.simplefilter('always')
```

Comment:
Just out of curiosity, is this still necessary with `pytest` or was this a holdover from before switching to `pytest`?

Reply:
Yes I find this is still necessary, some tests fail under PY2 without it.